substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/JDKLatest.java

```diff
@@ -28,9 +28,13 @@
 
 import jdk.graal.compiler.serviceprovider.JavaVersionUtil;
 
-public class JDK22OrEarlier implements BooleanSupplier {
+/**
+ * Denotes the latest supported JDK version. It corresponds to the highest key in the
+ * {@code JVMCI_MIN_VERSIONS} map in {@link jdk.graal.compiler.hotspot.JVMCIVersionCheck}.
```

Comment:
Would it make sense to move this map in a package that is accessible here and actually reference the highest key instead of hardcoding the check to `21`?

Reply:
The package is accessible, it is just the field that is `private`. The LTS version change so seldomly that IMHO it is fine to have a few references to it.